lib/valueSets.js

```diff
@@ -1,4 +1,5 @@
 const common = require('./common');
+const MVH = require('./multiVersionHelper');
```

Comment:
I think I've answered my earlier question about the value set exporter, above. The multiversion changes for value set exporter appear to be within the value set exporter, itself. Given that's the case, I think my question above can be disregarded.

Reply:
Too late.  Already answered. ;-)